receiver/prometheusreceiver/config.go

```diff
@@ -52,6 +52,10 @@ type Config struct {
 	// that requires that all keys present in the config actually exist on the
 	// structure, ie.: it will error if an unknown key is present.
 	ConfigPlaceholder interface{} `mapstructure:"config"`
+
+	// EnableProtobufNegotiation allows the collector to set the scraper option for
+	// protobuf negotiation when conferring with a prometheus client.
+	EnableProtobufNegotiation bool `mapstructure:"enable_protobuf_negotiation"`
```

Comment:
can you test this in one of the test cases (e.g. testdata/config.yaml) and config_test.go?